### PR TITLE
feat: add sass support

### PR DESCRIPTION
**Pull request by wearska**
Considering that all the styles end up inside the head tag of our html, I think it's safe to also generate sass files with new components, import them manually in the main "app.scss" file (located inside the "app" folder), which gets compiled to "app.css" and is linked in our index.html.

Also keep the generated .css files if anyone wants to use them

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/src/app.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// --------------------------------
+// Import all your other sass files below
+// Note: Order counts, so import your variables and mixins first
+// --------------------------------
+
+// e.g. @import 'app/components/example/example.scss'
```
Comment (IgorMinar) [inline addon/ng2/blueprints/ng2/files/src/app.scss]: listing every single scss file here is too error prone and besides, angular compiler will fetch all css files and prefix all selectors by default to provide css isolation. so we definitely don't want to list individual files here.

**Comment by IgorMinar** [comment]
We should support sass out of the box, but not like this.

- [x] transpile all scss files via the sass plugin as you have it
- [ ] do not concat all scss files via app.scss
- [ ] modify blueprints to generate scss file instead of css file for each component
- [ ] keep component blueprint requesting the css file (https://github.com/angular/angular-cli/blob/0f66aaca44e949d4b147f6c1bcf89348879a0c05/addon/ng2/blueprints/component/files/src/app/components/__name__/__name__.ts#L7)
- [ ] ensure that paths in the `dist` align so that Angular fetches the css file one compiling the component in the browser

a lot of this stuff will change once we move the template compilation to the build step, but for now the above should do

**Comment by wearska** [comment]
Yeah but, If you compile all .scss files in place you would need to point each file to the variables and mixins, if that file uses any of them.

**Comment by IgorMinar** [comment]
That sounds like a good thing to me. That's exactly what imports in the
typescript code do.

**Pull request closed without merging**